website/src/content/docs/getting-started/tutorial.md

```
@@ -177,3 +177,56 @@
+
+> [!NOTE] 
+> currently, you don't have access to shell specific values like `~` or `$HOME`
+> allthough, that may change in the future.
+
+##### example:
+```
+open /home/user/.config
+```
+
+#### `$ [your shell command here]`
+With the `$ ` command, any following inputs will be send to the shell.
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements
null

(MD038, no-space-in-code)